lib/shopify_transporter.rb

```diff
@@ -113,8 +121,9 @@ def run_json(file_name)
     file_data = File.read(file_name)
     parsed_file_data = Yajl::Parser.parse(file_data)
     return if parsed_file_data.nil? || parsed_file_data.empty?
-
+    raise_if_unexpected_json_structure(parsed_file_data)
```

Comment:
```ruby
raise UnexpectedJSONStructureError if unexpected_json_structure(parsed_file_data)

Reply:
👍 to Fahia's suggestion.

Reply:
this is actually even better than what i proposed. good shit